jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/federated/AccessLogForClassicLB.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright 2019 T Mobile, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ ******************************************************************************/
+/**
+  Copyright (C) 2019 T Mobile Inc - All Rights Reserve
+  Purpose:
+  Author :Avinash 
+  Date: Jan 21, 2019
+  
+ **/
+package com.tmobile.cloud.awsrules.federated;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.tmobile.cloud.awsrules.utils.PacmanUtils;
+import com.tmobile.cloud.constants.PacmanRuleConstants;
+import com.tmobile.pacman.commons.PacmanSdkConstants;
+import com.tmobile.pacman.commons.rule.BaseRule;
+import com.tmobile.pacman.commons.rule.PacmanRule;
+import com.tmobile.pacman.commons.rule.RuleResult;
+
+@PacmanRule(key = "check-for-access-log-for-classic-elb", desc = "checks for access log for application elb and s3 bucket name for access log", severity = PacmanSdkConstants.SEV_HIGH, category = PacmanSdkConstants.GOVERNANCE)
+public class AccessLogForClassicLB extends BaseRule {
+
+	private static final Logger logger = LoggerFactory.getLogger(AccessLogForClassicLB.class);
+
+	/**
+	 * The method will get triggered from Rule Engine with following parameters
+	 * 
+	 * @param ruleParam
+	 * 
+	 ************** Following are the Rule Parameters********* <br><br>
+	 * 
+	 *ruleKey : check-for-access-log-for-classic-elb <br><br>
+	 *
+	 *esAppElbWithInstanceUrl : Enter the application elb with instance api <br><br>
+	 *
+	 *threadsafe : if true , rule will be executed on multiple threads <br><br>
```

Comment:
If you are using any end points like "esAppElbWithInstanceUrl" then its recommend not to use in "threadsafe".
Please remove threadsafe.Since it hits the end point all the time, you may not get correct result if its in threadsafe as it runs in a mutithread

Reply:
All the review changes are incorporated.